### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project: lwgps
```
Comment (xansec) [inline Mayhemfile:1]: I recommend renaming your target to clarify what is being fuzzed.

File: fuzz/fuzz.c [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#include "lwgps/lwgps.h"
```
Comment (xansec) [inline fuzz/fuzz.c:1]: Perhaps we can rename this to `lwgps_init_fuzz`?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (ebell495) [reply Mayhemfile:1]: I've renamed it to fuzz_nmea_parser

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**